### PR TITLE
Update how-to doc to grant PROXY when replicating cluster from outside

### LINKED ISSUE #375: failed to grant PROXY privilege to moco-admin
**Describe the bug**

The following error occurs when creating a cluster that replicates data from an external mysqld according to [the user manual](https://cybozu-go.github.io/moco/usage.html#creating-a-cluster-that-replicates-data-from-an-external-mysqld).

```json
{"level":"error","ts":1645440573.910377,"logger":"grpc","caller":"server/clone.go:104","msg":"failed to initialize after clone","grpc.start_time":"2022-02-21T10:49:22Z","system":"grpc","span.kind":"server","grpc.service":"moco.Agent","grpc.method":"Clone","peer.address":"10.64.13.165:35340","error":"failed to grant to moco-admin: Error 1698: Access denied for user 'clone-init'@'localhost'"}
```

**Environments**
- Version: MOCO v0.10.9 and MySQL 8.0.18
- OS: Flatcar Container Linux (stable)

**To Reproduce**

1. Create moco-init user on the donor by `CREATE USER 'clone-init'@'localhost' IDENTIFIED BY 'deadbeef'` and `GRANT ALL ON *.* TO 'clone-init'@'localhost' WITH GRANT OPTION`
2. Create MySQLCluster
3. See error by `kubectl logs moco-${CLUSTER}-0 -c agent`

**Expected behavior**

Initialization on the recipient by moco-agent succeeds.

**Additional context**

This error occurs [here](https://github.com/cybozu-go/moco-agent/blob/348f568b0c88ad5330fc359b84968fd281402d02/server/initialize.go#L246). It was introduced by https://github.com/cybozu-go/moco-agent/pull/58.

In order to grant PROXY privileges to moco-admin user, it is not sufficient to grant ALL privileges to the clone-init user. The clone-init user must have PROXY privilege as well (and WITH GRANT OPTION to grant it to another user).

See https://dev.mysql.com/doc/refman/8.0/en/proxy-users.html#proxy-users-granting-proxy-privilege

### PR DESCRIPTION
**Pull request by morimoto-cybozu**
to fix #375

### CODE REVIEW

**Review by arosh — APPROVED** [review]
LGTM

File: docs/usage.md [hunk 1]
```diff
@@ -128,16 +128,19 @@ Let's call the source mysqld instance _donor_.
...
 ```console
 mysql> INSTALL PLUGIN clone SONAME mysql_clone.so;
 mysql> CREATE USER 'clone-donor'@'%' IDENTIFIED BY 'xxxxxxxxxxx';
 mysql> GRANT BACKUP_ADMIN, REPLICATION SLAVE ON *.* TO 'clone-donor'@'%';
 mysql> CREATE USER 'clone-init'@'localhost' IDENTIFIED BY 'yyyyyyyyyyy';
 mysql> GRANT ALL ON *.* TO 'clone-init'@'localhost' WITH GRANT OPTION;
+mysql> GRANT PROXY ON ''@'' TO 'clone-init'@'localhost' WITH GRANT OPTION;
 ```
 
+[^note]: The line of `GRANT PROXY ...` has been added for moco-agent >= 0.6.9 in response to [cybozu-go/moco-agent#58](https://github.com/cybozu-go/moco-agent/pull/58).
```
Comment (ymmt2005) [inline docs/usage.md:142]: This section describes a setup for cloning an external mysqld which is not necessarily a moco cluster.
So, I guess this note is misleading or at least is difficult to understand.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: docs/usage.md (hunk 1 above)
Comment (morimoto-cybozu) [reply docs/usage.md:142]: Well I'll delete this note.
What I wanted to say was "please don't miss the last line! It has been added!" so this note is just a transient advisory.

Comment (ymmt2005) [reply docs/usage.md]: So, shall we write that in the release note?

**Pull request merged**